src/components/SidePanel/HelpContent/helpContentMap.tsx

```diff
@@ -0,0 +1,731 @@
+                                    </Text>
+                                </>
+                            ),
+                        },
+                        ':scan': {
+                            content: ({styles}: {styles: ThemeStyles}) => (
+                                <>
+                                    <Text style={[styles.textHeadlineH1, styles.mb4]}>Scanned</Text>
+                                    <Text style={styles.textNormal}>A "scanned" expense was created by extracting the relevant details using the Concierge AI.</Text>
+                                </>
+                            ),
+                        },
+                        ':expensifyCard': {
+                            content: ({styles}: {styles: ThemeStyles}) => (
+                                <>
+                                    <Text style={[styles.textHeadlineH1, styles.mb4]}>Expensify Card</Text>
+                                    <Text style={styles.textNormal}>An "Expensify Card" expense corresponds to a "posted" (meaning, finalized by the bank) purchase.</Text>
+                                    <Text style={[styles.textNormal, styles.pt3]}>
+                                        Expensify Card expenses cannot be reimbursed as they are centrally paid by the bank account linked to the workspace.
+                                    </Text>
+                                </>
+                            ),
+                        },
+                        ':pendingExpensifyCard': {
+                            content: ({styles}: {styles: ThemeStyles}) => (
+                                <>
+                                    <Text style={[styles.textHeadlineH1, styles.mb4]}>Expensify Card (pending)</Text>
+                                    <Text style={styles.textNormal}>
+                                        A "pending" Expensify Card expense represents a purchase that was recently made on the card, but has not yet "posted" – meaning, it has not been
+                                        formally recognized as a final, complete transaction.
+                                    </Text>
+                                    <Text style={[styles.textNormal, styles.pt3]}>Any changes made to this expense will be preserved when the expense posts, typically 2-7 days later.</Text>
+                                    <Text style={[styles.textNormal, styles.pt3]}>
+                                        Pending transactions cannot be approved, as the final expense amount will not be confirmed until it posts.
+                                    </Text>
+                                </>
+                            ),
+                        },
+                    },
+                },
+            },
+        },
+        home: {
+            content: ({styles}: {styles: ThemeStyles}) => (
```

Comment:
This whole function returns the same thing as at the root (lines 30-148). Thoughts on reusing a const that's defined beforehand instead? We don't necessarily have to do it now, but as it stands I'm thinking that down the line, we might want to edit the copy and risk forgetting to edit both places at the same time.

Reply:
I wanted to keep it simple due to the fact this file is going to be generated going forward, but I totally agree with you.